src/solvers/README.md

```diff
@@ -228,7 +228,44 @@ To be documented.
 
 \subsection sat-solving-api-section SAT solving API
 
-To be documented.
+The basic SAT solver interface is in [cnf](\ref cnft). This inherits
+from the [propositional logic decision procedure wrapper](\ref propt).
+
+The interface supports the following operations by default:
+1. Boolean operations on literals (like `and`, `or`, `xor`), etc.
+These take as input two [literal](\ref literalt) and return as
+output another [literal](\ref literalt).
```

Comment:
This should likely reference https://en.wikipedia.org/wiki/Tseytin_transformation or some more original description of it.